### PR TITLE
Let vertex and edge adapt simple json properties

### PR DESCRIPTION
**Pull request by Linary**
Implement #36

Change-Id: I500976931c50fbab384dbd71dbd839aa69dc42d3

### CODE REVIEW

File: src/main/java/com/baidu/hugegraph/driver/HugeClient.java [hunk 1]
```diff
@@ -96,7 +96,7 @@ private void initManagers(RestClient client, String graph) {
     private void checkServerApiVersion() {
         VersionUtil.Version apiVersion = VersionUtil.Version.of(
                                          this.version.getApiVersion());
-        VersionUtil.check(apiVersion, "0.32", "0.33",
+        VersionUtil.check(apiVersion, "0.34", "0.35",
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/driver/HugeClient.java]: why skip api version 0.33

File: src/main/java/com/baidu/hugegraph/serializer/VertexDeserializer.java [hunk 2]
```diff
@@ -17,7 +17,7 @@
 
     private final ObjectMapper mapper;
 
-    public VertexDeserializer() {
+    private VertexDeserializer() {
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/serializer/VertexDeserializer.java]: add some comments

File: src/main/java/com/baidu/hugegraph/driver/HugeClient.java (hunk 1 above)
Comment (Linary) [reply src/main/java/com/baidu/hugegraph/driver/HugeClient.java]: 0.33 is customized path api

File: src/test/java/com/baidu/hugegraph/unit/RestResultTest.java [hunk 3]
```diff
@@ -409,54 +404,30 @@ public void testReadVertices() {
...
-                + "\"value\": \"peter\"" 
-                + "}],"
-                + "\"age\": [{"
-                + "\"id\": \"person:peter>age\"," 
-                + "\"value\": 29" 
-                + "}]}"
+                + "\"city\": \"Shanghai\","
+                + "\"name\": \"peter\","
+                + "\"age\": 29"
+                + "}"
```
Comment (javeme) [inline src/test/java/com/baidu/hugegraph/unit/RestResultTest.java]: add test for list/set property

Comment (Linary) [reply src/test/java/com/baidu/hugegraph/unit/RestResultTest.java]: now list/set property will be parsed into a list

**Pull request merged**